### DISCUSSION TITLE
狗臉辨識!     寵物保險「刷臉理賠」辨識應用新藍海

### DISCUSSION THREAD

**Discussion by yi-1111** [1101-GS4719-C/Python-Community-GS4719-C#432] [科技新知]
人臉辨識已經是市場上相當成熟的技術，包括手機解鎖、辦公室門禁系統、機場通關、線上金融服務都能應用得到。
現今也應用在寵物身上了，心愛的寵物生病了，獸醫師拍張照，後台系統比對註冊照片和晶片資料，確認是曾經投保的寵物，保險公司才啟動理賠程序，疫情期間，線上保險、線上貸款愈來愈普及，新創公司結合人臉偵測，影音偵測詐欺等多重防護，在授權與金融交易扮演重要角色。

https://news.tvbs.com.tw/tech/1726933

**Comment by pekoranoinu** [comment]
少子化的社會，寵物的地位也變得越來越高。

**Comment by chupochun** [comment]
連寵物都可進行臉部辨識了，人工智慧實在是太廣大了

**Comment by sean1325418** [comment]
其實人的臉部辨識也就只是把圖片丟入程式中(可能是CNN)，讓他自己尋找特徵。這樣訓練完之後就能辨識之後的圖片了。既然人臉可以那當然狗臉也可以，就算是貓也都沒什麼問題。

**Comment by awkward-willy** [comment]
寵物相對來說較難控制
很難剛好在鏡頭前辨識的感覺
也很好奇會不會被寵物的毛髮等因素影響
（畢竟他們毛髮量較人類多很多，可能會遮住重要辨識點）

**Comment by Rosancl** [comment]
還滿好奇寵物辨識的辨識點跟人臉辨識有甚麼差異

**Comment by Lai27330** [comment]
感覺寵物較難配合臉部辨識，而且相較於人臉更容易有毛髮長短等較大的變化，因此感覺辨識點更難掌握，但對著鏡頭揮手打卡感覺好酷!

**Comment by treeann** [comment]
狗臉辨識也太強了八 感覺比人臉辨識更加困難

**Comment by Yuhsin000** [comment]
可以辨識狗狗聽起來很可愛，困難點應該在於毛髮的部分(?

**Comment by leon10456** [comment]
感謝樓主的分享

**Comment by Yao900108** [comment]
原來辨識系統也可以應用在動物的臉上，真有趣!

**Comment by liangyinglly** [comment]
之前學圖像辨識的時候有學到可以利用KNN並把維度提高應該可以找到一個平面輕易的區分狗狗與貓貓 個技術應該是類似的技巧吧

**Comment by stan890425** [comment]
我們家狗狗的指紋也可以解鎖我的手機~~

**Comment by jcpan821** [comment]
但是感覺狗勾狀臉的機率很高

**Comment by alexthebigpp** [comment]
人類肉眼有時候分不太出來，靠機器就可以讓事情方便很多

**Comment by Ben6uo** [comment]
確實養寵物比養小孩便宜上不少 還不會發脾氣

**Comment by yenwei12138** [comment]
在現代人養小孩不如養寵物的想法下，可能使用率會漸漸上升

**Comment by Lynn12378** [comment]
狗狗們的長相感覺都很相似，能辨認出來的話也太強了

**Comment by etin2468** [comment]
果然毛小孩的市場很大，連AI都應用上了

**Comment by 110409538** [comment]
那之後寵物是不是也可以線上看診

**Comment by Hchejxnc** [comment]
寵物都可進行臉部辨識了，AI能運用的地方真的太廣了

**Comment by CPCpyt** [comment]
非常期待這個技術延伸到其他種類的動物，
或對於寵物的關照上也許能有進一步的幫助。

**Comment by harrylee1971** [comment]
以後這樣狗狗們可不可以變成存款的工具，因為可以刷臉，代表有隱私性的擁有性了
